simulator.js

```diff
@@ -274,21 +274,7 @@ class Battle {
 	}
 	onRename(user, oldid) {
```

Comment:
Oh I guess it already is. Why don't we get rid of the old `onRename` and consolidate it under one?